go/host/host.go

```diff
@@ -102,7 +113,27 @@ func NewHost(
 	}
 
 	if config.HasClientRPCHTTP || config.HasClientRPCWebsockets {
-		host.rpcServer = NewRPCServer(config, host)
+		rpcAPIs := []gethrpc.API{
```

Comment:
The separation of concerns is better if the RPC server is provided a list of RPC APIs to install, rather than having them hardcoded.